Applications/Calculator/CalculatorWindow.gml

```diff
@@ -0,0 +1,275 @@
+            text: "6"
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "blue"
+        }
+
+        @GUI::Button {
+            name: "multiply_button"
+            text: "*"
+            fixed_width: 35
+            fixed_height: 28
+        }
+
+        @GUI::Button {
+            name: "mod_button"
+            text: "%"
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "blue"
+        }
+    }
+
+    @GUI::Widget {
+
+        layout: @GUI::HorizontalBoxLayout {}
+
+        @GUI::Button {
+            name: "mem_save_button"
+            text: "MS"
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "red"
+        }
+
+        @GUI::Widget {
+            fixed_width: 5
+        }
+
+        @GUI::Button {
+            name: "1_button"
+            text: "1"
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "blue"
+        }
+
+        @GUI::Button {
+            name: "2_button"
+            text: "2"
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "blue"
+        }
+
+        @GUI::Button {
+            name: "3_button"
+            text: "3"
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "blue"
+        }
+
+        @GUI::Button {
+            name: "subtract_button"
+            text: "-"
+            fixed_width: 35
+            fixed_height: 28
+        }
+
+        @GUI::Button {
+            name: "inverse_button"
+            text: "1/x"
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "blue"
+        }
+    }
+
+    @GUI::Widget {
+
+        layout: @GUI::HorizontalBoxLayout {}
+
+        @GUI::Button {
+            name: "mem_add_button"
+            text: "M+"
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "red"
+        }
+
+        @GUI::Widget {
+            fixed_width: 5
+        }
+
+        @GUI::Button {
+            name: "0_button"
+            text: "0"
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "blue"
+        }
+
+        @GUI::Button {
+            name: "sign_button"
+            text: "+/-"
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "blue"
+        }
+
+        @GUI::Button {
+            name: "decimal_button"
+            text: "."
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "blue"
+        }
+
+        @GUI::Button {
+            name: "add_button"
+            text: "+"
+            fixed_width: 35
+            fixed_height: 28
+        }
+
+        @GUI::Button {
+            name: "equal_button"
+            text: "="
+            fixed_width: 35
+            fixed_height: 28
+            foreground_color: "red"
+        }
+    }
+}
+
```

Comment:
I think you added two newlines :sweat_smile:

Reply:
This is frustrating, added 1 and it didn't show here on GitHub now added 2 and it showed 1

Reply:
@emanuele6  How do you see the additional newline? guthub ui only shows 1

Reply:
"newline" refers to a "newline character" really - when GH shows a missing newline indicator there's none (`...\n}EOF`), if it shows the last line normally there's one (`...\n}\nEOF`), but if there's an additional empty line there's two (`...\n}\n\nEOF`. like here).

Reply:
oh so based on that i was fine the first time

Reply:
removed the extra line

Reply:
> oh so based on that i was fine the first time

No, github was showing this symbol before
![file](https://user-images.githubusercontent.com/20175435/103481214-c4354700-4dd9-11eb-946e-1fb391952f57.png)

If it shows a symbol like that, it means that you had `...\n}EOF`.

Now you have `...\n}\nEOF`.